### PR TITLE
universal-ctags: New port

### PR DESCRIPTION
**Pull request by kaimingguo**
###### Description

universal-ctags has the objective of continuing the development from what existed in the Sourceforge area. Github exuberant-ctags repository was started by Reza Jelveh and was later moved to the universal-ctags organization.

The goal of the project is preparing and maintaining common/unified space where people interested in making ctags better can work together.

###### Type(s)

- [ ] bugfix
- [x] enhancement
- [ ] security fix

###### Tested on

macOS 10.15.2 19C57
Xcode 11.3 11C29

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL?

- [x] checked your Portfile with `port lint`?
- [ ] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

**Comment by g5pw** [comment]
This looks good to me, the only question that pops to mind is why does this use python3.7 and not 3.8 (the latest version)?

**Comment by mf2k** [comment]
I agree with with Kai-Ming and think we should stick to python 3.7 until we are ready to switch over to python 3.8 for all ports that require python. We are still waiting for some py38 ports to be created.

**Comment by g5pw** [comment]
Ok, that's fine by me. One last thing, we tend to not add unmaintained ports (since there are a lot of them already). Would you like to take maintainership, @kaimingguo?

**Comment by kaimingguo** [comment]
> Ok, that's fine by me. One last thing, we tend to not add unmaintained ports (since there are a lot of them already). Would you like to take maintainership, @kaimingguo?

"nomaintainer" was originally used so that everyone else could submit updates, I'd be happy to take maintainership for me. I update maintainer now. :)

**Comment by g5pw** [comment]
Well, if you want to allow everyone to update, you can use *open* maintainership by adding `openmaintainer`  to the maintainers line. This will signal that making small changes without running them past you (the maintainer) is okay.

Add that line and squash all your commits together and I'll merge 😊

File: devel/universal-ctags/Portfile [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+PortGroup           github 1.0
+
+github.setup        universal-ctags ctags feffe43a4a9d339cd6701744589746e2297e4567
+name                universal-ctags
+version             20200111
+
+categories          devel
+platforms           darwin
+license             GPL-2+
+maintainers         {@kaimingguo gmail.com:augustin.guo}
```
Comment (mf2k) [inline devel/universal-ctags/Portfile]: Can `openmaintainer` be added?

**Comment by mf2k** [comment]
Please squash the commits.

**Comment by kaimingguo** [comment]
Squash commits and add `openmaintainer` in `Portfile`.

**Comment by g5pw** [comment]
Thank you for your contribution to Macports @kaimingguo! Merging :)

**Pull request merged**